app/views/admin/lead_providers/review.html.erb

```diff
@@ -0,0 +1,72 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: admin_new_lead_provider_cohorts_path)
+%>
+
+
+
```

Comment:
let's not have more than one empty line in views? I think you can globally search for double endlines in rubymine